contrib_bots/bots/weather/weather.py

```diff
@@ -0,0 +1,74 @@
+# See readme.md for instructions on running this code.
+from __future__ import print_function
+import requests
+import json
+import os
+import sys
+from six.moves.configparser import SafeConfigParser
+
+DIRECTORY = os.path.dirname(os.path.realpath(__file__)) + '/'
+CONFIG_FILE = '.weather_config'
+RESPONSE_PATTERN = 'Weather in {}, {}:\n{} F / {} C\n{}'
+
+if not os.path.exists(DIRECTORY + CONFIG_FILE):
+    print('Weather bot config file not found, please set it up in {} file in this bot main directory'
+          '\n\nUsing format:\n\n[weather-config]\nkey=<OpenWeatherMap API key here>\n\n'.format(CONFIG_FILE))
+    sys.exit(1)
```

Comment:
This is fine for now, but in general throwing an exception (or returning an error) is better than `sys.exit(1)` because we don't want functions that we call to be able to randomly just crash the entire program.